pkg/template/evaluation_ctx.go

```diff
@@ -88,6 +89,21 @@ func (e *EvaluationCtx) TplSetNode(thread *starlark.Thread, _ *starlark.Builtin,
 	return starlark.None, nil
 }
 
+// args(nodeTag, value Value)
+func (e *EvaluationCtx) TplSetMapItemKey(thread *starlark.Thread, _ *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	if args.Len() > 1 {
+		if _, noop := args.Index(1).(*core.StarlarkNoop); !noop {
```

Comment:
dont think we need to worrry about StarlarkNoop here. we need it for TplSetNode because of `template.replace` usage.